src/IO/SeekAvoidingReadBuffer.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <IO/ReadBufferFromFileBase.h>
+
+
+namespace DB
+{
+
+class SeekAvoidingReadBuffer : public ReadBufferFromFileBase
```

Comment:
Ok... but better to add a comment nevertheless.

Reply:
+